app/src/main/java/nl/tudelft/bbw/crawler/BlockDatabase.java

```diff
@@ -0,0 +1,73 @@
+package nl.tudelft.bbw.crawler;
+
+import android.content.Context;
+import android.database.sqlite.SQLiteDatabase;
+
+import com.readystatesoftware.sqliteasset.SQLiteAssetHelper;
+
+/**
+ * Database handler for the block database implementation of the crawler
+ * https://github.com/YourDaddyIsHere/even-cleaner-neighbor-discovery/blob/master
+ * /HalfBlockDatabase.py
+ */
+public class BlockDatabase extends SQLiteAssetHelper {
```

Comment:
This name is confusing; what is difference between our own blocks in the database and wherever this class looks?